### DISCUSSION TITLE
Trigger test button does not renew data unless account is removed and logged in again

### DISCUSSION THREAD

**Discussion by hanifaudah** [bahasa-ai/otomatis-guidance#25] [General]
The "Test Trigger" button does not renew the data unless the test account is renewed
  Steps to reproduce:
  - Create Trigger
  - Press "Test Trigger" button
  - Change response in perform function
  - Press "Test Trigger" button

Actual: Data is the same
Expected: Data changes
![image](https://user-images.githubusercontent.com/56548631/129851172-670aff44-2c41-4ea8-93a9-8396a9d26993.png)

**Comment by mgilangjanuar** [comment]
https://user-images.githubusercontent.com/3146378/130002309-4db6f204-e012-48a4-8e2d-52f975903df4.mov

Hmm I think you skip to click the *save button*, aren't you? 🤔
